docs/eessi.md

```diff
@@ -0,0 +1,63 @@
+# EESSI Guide
+
+## How to load EESSI
+Loading an EESSI environment module:
+```[bash]
+source /cvmfs/software.eessi.io/versions/2023.06/init/lmod/bash
+```
+Activating EESSI environment:
+```[bash]
+source /cvmfs/software.eessi.io/versions/2023.06/init/bash
+```
+
+## GPU Support with EESI
+To enable GPU support you need a site-specific build that has `cuda` enabled. For guide to do this please refer to `docs/image-build`. This is
+because CUDA-drivers are host specific and EESSI can not ship NVIDIA drivers due to licensing + kernel specific constraints. This means that the
```

Comment:
typo "can not" -> "cannot" but also; they could ship the open drivers, but they don't. They cannot ship all the CUDA packages. So the situation is complex and probably more accurately explained on their page TBH.

So actually - I'd just remove the last two sentences in this para.